lib/tasks/deleted_objects.rake

```diff
@@ -11,4 +11,16 @@ namespace :deleted_objects do
       sleep(0.5) unless Rails.env.test?
     end
   end
+
+  desc 'Destroy all the service objects and those whose owner is also a deleted object. Objects destroyed at least 1 week ago.'
```

Comment:
I am thinking that what we should delete here are the objects whose owner is a Service + the services themselves 🤔 And at this point, it won't matter if it was 1 week ago or 1 minute ago because in the code, the won't be used anymore anyway 🤔